test/powershell/Modules/Microsoft.PowerShell.Utility/Update-FormatData.Tests.ps1

```diff
@@ -57,3 +57,50 @@ Describe "Update-FormatData basic functionality" -Tags "CI" {
         { Update-FormatData -Prepend $testfile -WhatIf } | Should Not Throw
 	}
 }
+
+
+Describe "Update-FormatData with resources in CustomControls" -Tags "CI" {
+

```

Comment:
The file has multiple wrong indentations. See raw view https://raw.githubusercontent.com/anmenaga/PowerShell/fd95dd19b90e7cb5d0094f52503b59c59ec1ca5b/test/powershell/Modules/Microsoft.PowerShell.Utility/Update-FormatData.Tests.ps1
It seems you copy-paste tabs.
Could you please reformat by first commit and put new code in next commits?